modules/migration-mtc-release-notes-1-8-1.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-1_{context}"]
+= {mtc-full} 1.8.1 release notes
+
+[id="resolved-issues-1-8-1_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+.CVE-2023-39325: golang: net/http, x/net/http2: rapid stream resets can cause excessive work
+
+A flaw was found in handling multiplexed streams in the HTTP/2 protocol, which is utilized by {mtc-full} ({mtc-short}). A client could repeatedly make a request for a new multiplex stream and immediately send an `RST_STREAM` frame to cancel it. This creates additional workload for the server in terms of setting up and dismantling streams, while avoiding any server-side limitations on the maximum number of active streams per connection, resulting in a denial of service due to server resource consumption. link:https://bugzilla.redhat.com/show_bug.cgi?id=2245079[(BZ#2245079)]
+
+It is advised to upgrade to {mtc-short} 1.8.1 or later, which resolve this issue.
```

Comment:
Per our term glossary, [update is preferred over upgrade](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update).
```suggestion
It is advised to update to {mtc-short} 1.8.1 or later, which resolve this issue.
```